src/firmware_new/boards/arduino_uno/BUILD

```diff
@@ -0,0 +1,23 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@platformio_rules//platformio:platformio.bzl",
+    "platformio_library",
+    "platformio_project",
+)
+
+platformio_library(
+    name = "echo_library",
+    src = "echo_library.cc",
+    hdr = "echo_library.h",
+)
+
+platformio_project(
```

Comment:
does running this flash the arduino? A bit of documentation would help

Reply:
running the platformio_project flashes the arduino, will add more documentation